src/elements/radio-button/radio-button-group/radio-button-group.visual.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { html, type TemplateResult } from 'lit';
+import { styleMap, type StyleInfo } from 'lit/directives/style-map.js';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualDiffFocus,
+} from '../../core/testing/private.js';
+
+import '../../card/card-badge.js';
+import '../../form-error.js';
+import '../../icon.js';
+import '../../radio-button.js';
+
+const cases = {
+  disabled: [false, true],
+  orientation: ['vertical', 'horizontal'],
+  size: ['m', 's'],
+};
+
+const suffixStyle: Readonly<StyleInfo> = {
+  display: 'flex',
+  alignItems: 'center',
+  marginInline: 'var(--sbb-spacing-fixed-2x)',
+};
+
+const suffixAndSubtext = (): TemplateResult => html`
+  <span slot="subtext">Subtext</span>
+  <span slot="suffix" style="margin-inline-start: auto;">
+    <span style=${styleMap(suffixStyle)}>
+      <sbb-icon name="diamond-small" style="margin-inline: var(--sbb-spacing-fixed-2x);"></sbb-icon>
+      <span class="sbb-text-m sbb-text--bold">CHF 8.00</span>
+    </span>
+  </span>
+  <sbb-card-badge>%</sbb-card-badge>
+`;
+
+const radioButtons = (): TemplateResult => html`
+  <sbb-radio-button value="Value one">Value one</sbb-radio-button>
+  <sbb-radio-button value="Value two">Value two</sbb-radio-button>
+  <sbb-radio-button value="Value three" disabled> Value three </sbb-radio-button>
+  <sbb-radio-button value="Value four">Value four</sbb-radio-button>
+`;
+
+const radioButtonPanels = (): TemplateResult => html`
+  <sbb-radio-button-panel value="Value one">Value 1 ${suffixAndSubtext()}</sbb-radio-button-panel>
+  <sbb-radio-button-panel value="Value two">Value 2 ${suffixAndSubtext()}</sbb-radio-button-panel>
+`;
+
+const variants: { name: string; template: TemplateResult }[] = [
+  { name: 'radio-button', template: radioButtons() },
+  { name: 'radio-button-panel', template: radioButtonPanels() },
+];
+
+describe(`sbb-radio-button-group`, () => {
+  describeViewports({ viewports: ['small', 'medium'] }, () => {
+    describeEach(cases, ({ orientation, size, disabled }) => {
+      for (const variant of variants) {
+        for (const state of [visualDiffDefault, visualDiffFocus]) {
+          (disabled && state === visualDiffFocus ? it.skip : it)(
+            `${variant.name} ${state.name}`,
```

Comment:
not a problem, but could also have a `describe(variant.name)` directly below `for (const variant of variants) {`